### PR TITLE
reference/tidb-binlog: update aliases in monitor v3.0

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? 

Update aliases in monitor.

This PR includes updates that only apply to v3.0.
Updates that also apply to dev and v3.1 are made in this PR: https://github.com/pingcap/docs/pull/1881

### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with: https://github.com/pingcap/docs-cn/pull/2224
- [ ] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

**Comment by ran-huang** [comment]
@TomShawn Not sure if I put the aliases right. PTAL😀

File: reference/tidb-binlog/monitor.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: TiDB Binlog Monitoring
 summary: Learn how to monitor the cluster version of TiDB Binlog.
 category: reference
-aliases: ['/docs/tools/binlog/monitor/','/docs/v3.0/how-to/monitor/tidb-binlog/']
+aliases: ['/docs/tools/binlog/monitor/','/docs/v3.0/how-to/monitor/tidb-binlog/','/docs/dev/how-to/monitor/tidb-binlog/']
```
Comment (TomShawn) [inline reference/tidb-binlog/monitor.md]:
```suggestion
aliases: ['/docs/dev/how-to/monitor/tidb-binlog/']
```

Comment (TomShawn) [inline reference/tidb-binlog/monitor.md]: As tested, this seems to the only available alias.
```suggestion
aliases: ['/docs/tools/binlog/monitor/']
```

Comment (ran-huang) [reply reference/tidb-binlog/monitor.md]: Many thanks!

**Pull request merged**